.gitignore

```diff
@@ -115,4 +115,5 @@ dmypy.json
 .pyre/
 
 # Mac OS
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode/settings.json
```

Comment:
Don't put .vscode/settings.json in gitignore. 

If there are conflicts in `pythonPath`, you can define an env variable and put that in ~/.bashrc or ~/.zshrc. 

Something like this:
```
  "python.pythonPath": "${env.JOVIAN_PY_PYTHON}",
```